exporters/zipkin/src/recordable.cc

```diff
@@ -0,0 +1,184 @@
+#include "opentelemetry/exporters/zipkin/recordable.h"
+
+#include <iostream>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+
+const int kAttributeValueSize = 14;
+
+void Recordable::SetIds(trace::TraceId trace_id,
+                        trace::SpanId span_id,
+                        trace::SpanId parent_span_id) noexcept
+{
+  char traceid[trace::TraceId::kSize * 2] = {0};
```

Comment:
Inconsistent naming for `traceid`, probably `trace_id_lower_base16`?

Reply:
Good comment. Have changed for all three id's.